paths/graphql/06-mutation.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Create a Mutation
+permalink: /graphql/mutation
+facilitator: false
+sidebar:
+  nav: "github-graphql"
+main-content: |
+
+  So far, we've only used GraphQL to form **queries**, or requests for information. Next, we're going to use the information we've gathered to craft a **mutation**. Simply put, a mutation changes information instead of just consuming it.
+
+  1. Get your information from the both previous queries ready (list of PRs, and the issue's `id`).
+  1. Create a mutation by either typing or pasting the following into the [GraphQL explorer](https://developer.github.com/v4/explorer/). Change `[issueID]` to the issue `id` you found in the most recent query, and `[PR template]` to the template you completed earlier based on your most recent pull requests. _You will need to delete the new lines in between the list items in the template for it to process correctly._
+
+          mutation AddComment {
+            addComment(input: {
+              subjectId: "[issueID]",
+              body: "[PR template]"})
+            {
+              subject {
+                id
+              }
+            }
+          }
+
+        > Look around using the built in documentation. By hovering, erasing, retyping, or changing fields, what can you learn about this mutation?
+
+   1. Go to the [issue](https://github.com/githubschool/graph-ql/issues/1) and find your posted comment.
```

Comment:
Agreed - I think we should include something similar to [we have in the intro course](https://services.github.com/on-demand/intro-to-github/join-class-repository). Ideally we could even re-use this code. I'll ✋ this now.